Source/engine/dx.cpp

```diff
@@ -117,7 +117,7 @@ void dx_cleanup()
 #ifndef USE_SDL1
 	texture = nullptr;
 	FreeVirtualGamepadTextures();
-	if (*GetOptions().Graphics.upscale)
+	if (false)

```

Comment:
Should also be DOS-only, or even better would be disable this option for DOS (similar to how we disable certain options for other platforms)